berkeley-mobile/Common/EventTableViewCell.swift

```diff
@@ -78,9 +84,15 @@ class EventTableViewCell: UITableViewCell {
         eventName.setContentHuggingPriority(.required, for: .vertical)
         eventName.leftAnchor.constraint(equalTo: contentView.layoutMarginsGuide.leftAnchor).isActive = true
         eventName.topAnchor.constraint(equalTo: contentView.layoutMarginsGuide.topAnchor).isActive = true
-        eventName.rightAnchor.constraint(equalTo: contentView.layoutMarginsGuide.rightAnchor).isActive = true
+        eventName.rightAnchor.constraint(equalTo: eventLogo.leftAnchor, constant: 15).isActive = true
```

Comment:
I think this should be a constant of -15 or something if you want spacing between the two items.

Reply:
Oops!